src/WebJobs.Script.WebHost/Configuration/ScriptHostOptionsSetup.cs

```diff
@@ -0,0 +1,255 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Script.WebHost.Configuration;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    internal class ScriptHostOptionsSetup : IConfigureOptions<ScriptHostOptions>
```

Comment:
Belongs in WebJobs.Script?

Reply:
We could move this to WebJobs.Script. A lot of configuration things landed here as a lot of it is host specific.

Reply:
Yes - for these new types we're adding in this PR we should maintain the layering that we've had in place. Non WebHost specific stuff is pushed into WebJobs.Script (i.e. most everything).